GoogleForCreators/web-stories-wp
# Design System: Move `useLiveRegion` to design system and fix Snackbar stories

## Context

Preparing to add `useLiveRegion` to many parts of the codebase.

## Summary

We are going to add `useLiveRegion` to many things in the codebase. This PR:
- moves the `useLiveRegion` hook from the editor to the design system
- fixes the storybook of the snackbar (need this so we can test in storybook with voiceover)
- cleans up the snackbar api a little bit for the stories. None of this is user facing.

## Relevant Technical Choices

Didn't want this PR to get to big, so the next pr will actually implement `useLiveArea` all over the place.

## To-do

Need to add `useLiveRegion` to many things (some not even in the ticket 😱).

## User-facing changes

n/a

## Testing Instructions

1. Go to storybook, make sure the snackbar stories work. Click knobs, see actions!

### QA






### UAT

n/a

## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Partially addresses #6059
